src/jit/SimdArm32Inl.h

```diff
@@ -109,6 +123,315 @@ enum ExtendType : uint32_t {
 
 }; // namespace SimdOp
 
+using unaryCallbackFunction = std::add_pointer<void(void*, void*)>::type;
+using binaryCallbackFunction = std::add_pointer<void(void*, void*, void*)>::type;
+
+static void prepareF32(void* src0, float** src0Ptr, float src0Storage[4], void* src1, float** src1Ptr, float src1Storage[4], void* dst, float** dstPtr)
+{
+    if (LIKELY((unsigned long)src0 & 15) == 0) {
+        *src0Ptr = reinterpret_cast<float*>(src0);
+    } else {
+        *src0Ptr = src0Storage;
+        memcpy(src0Storage, src0, 4 * sizeof(float));
+    }
+
+    if (src1 != nullptr) {
+        if (LIKELY(((unsigned long)src1 & 15) == 0)) {
+            *src1Ptr = reinterpret_cast<float*>(src1);
+        } else {
+            *src1Ptr = src1Storage;
+            memcpy(src1Storage, src1, 4 * sizeof(float));
+        }
+    }
+
+    *dstPtr = reinterpret_cast<float*>(dst);
```

Comment:
Is there a big gain here? Perhaps the `void* dst` arguments could simply be `float* dstPtr`, since they are always aligned